### PR TITLE
milkvisits patch

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/usaco-968.mdx [hunk 1]
```diff
@@ -2,51 +2,38 @@
 id: usaco-968
 source: USACO Silver 2019 December
 title: Milk Visits
-author: Qi Wang, Tanish Tyagi
+author: Qi Wang, Tanish Tyagi, Kevin Sheng
 ---
 
-[Official Analysis](http://www.usaco.org/current/data/sol_milkvisits_silver_dec19.html)
+[Official Analysis (c++)](http://www.usaco.org/current/data/sol_milkvisits_silver_dec19.html)
```
Comment (nwatx) [inline solutions/usaco-968.mdx:8]: c++ or C++

File: solutions/usaco-968.mdx [hunk 2]
```diff
@@ -2,51 +2,38 @@
...
 **Time Complexity:** $\mathcal{O}(N + M)$
 
-We can use a DFS to create connected components from an adjacency list (which is
-made when we process the roads between the farms). Each connected component will
-contain farms that all have the same breed.
+We perform a search on the tree, identifying connected components of cows that
+all have the same breed.
 
-Adjacency List based off sample test case:
+In the sample test case, there's $3$ connected components:
```
Comment (nwatx) [inline solutions/usaco-968.mdx:15]:
```suggestion
In the sample test case, there are $3$ connected components:
```

File: solutions/usaco-968.mdx [hunk 3]
```diff
@@ -59,216 +46,213 @@ Queries for Sample Test Case:
...
+	for (char& f : farms) {
+		read >> f;
+		assert(f == 'G' || f == 'H');
+	}
+
+	vector<vector<int>> neighbors(farm_num);
+	for (int f = 0; f < farm_num - 1; f++) {
+		int f1, f2;
+		read >> f1 >> f2;
+		neighbors[--f1].push_back(--f2);
```
Comment (nwatx) [inline solutions/usaco-968.mdx:78]: would be more clear if decrements were done in line above

**Review by nwatx — APPROVED** [review]
other than these, looks good

File: solutions/usaco-968.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/usaco-968.mdx:8]:
```suggestion
[Official Analysis (C++)](http://www.usaco.org/current/data/sol_milkvisits_silver_dec19.html)
```

File: solutions/usaco-968.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply solutions/usaco-968.mdx:15]: uh could you elaborate? both seem non-awkward tbh

Comment (nwatx) [reply solutions/usaco-968.mdx:15]: the first one is grammatically wrong (i think)

**Review by ryanchou-dev — APPROVED** [review]
great work team! >v<

**Pull request merged**